### PR TITLE
NFT page responsive tweak

### PR DESCRIPTION
**Pull request by haolinj**

### CODE REVIEW

File: code/client/src/components/NFTGrid.jsx [hunk 1]
```diff
@@ -73,22 +71,31 @@ const NFTGridItem = ({ disabled, style, styleFullView, imageWrapperStyle, imageW
...
         <Row justify='space-between' style={{ padding: 8 }}>
-          {metadata && <Text style={{ fontSize: 12, lineHeight: '16px' }}>{displayName}</Text>}
-          {!metadata &&
-            <Text
-              style={{ fontSize: 12 }}
-              copyable={{ text: abbrBech32ContractAddress }}
-            >{util.ellipsisAddress(abbrBech32ContractAddress)}
-            </Text>}
-          <Text style={{ fontSize: 12, lineHeight: '16px' }}>{displayBalance}</Text>
+          <Col span={12}>
```
Comment (polymorpher) [inline code/client/src/components/NFTGrid.jsx:87]: should not use two columns. Row is a flex box, the code was using space-between to intentionally positioning the two child nodes at beginning and end sides

File: code/client/src/components/NFTGrid.jsx [hunk 2]
```diff
@@ -73,22 +71,31 @@ const NFTGridItem = ({ disabled, style, styleFullView, imageWrapperStyle, imageW
   return (
     <GridItem style={fullView ? styleFullView : style} hoverable={false} onClick={() => !fullView && interactable && setFullView(true)} data-full-view={fullView}>
       {!fullView &&
-        <Row style={{ height: wrapperStyle.height || 'auto' }}>
-          <Image
-            preview={false} src={util.replaceIPFSLink(metadata?.image) || FallbackImage} fallback={FallbackImage}
-            wrapperStyle={wrapperStyle} style={{ objectFit: 'cover', width: '100%', height: '100%' }}
-          />
+        <Row>
+          <Col span={24}>
```
Comment (polymorpher) [inline code/client/src/components/NFTGrid.jsx:75]: no col

File: code/client/src/components/NFTGrid.jsx [hunk 3]
```diff
@@ -73,22 +71,31 @@ const NFTGridItem = ({ disabled, style, styleFullView, imageWrapperStyle, imageW
   return (
     <GridItem style={fullView ? styleFullView : style} hoverable={false} onClick={() => !fullView && interactable && setFullView(true)} data-full-view={fullView}>
       {!fullView &&
-        <Row style={{ height: wrapperStyle.height || 'auto' }}>
-          <Image
-            preview={false} src={util.replaceIPFSLink(metadata?.image) || FallbackImage} fallback={FallbackImage}
-            wrapperStyle={wrapperStyle} style={{ objectFit: 'cover', width: '100%', height: '100%' }}
-          />
+        <Row>
```
Comment (polymorpher) [inline code/client/src/components/NFTGrid.jsx:74]: need wrapperStyle to keep the heights uniform for all items in the grid across multiple rows

**Comment by polymorpher** [comment]
I will check the UI later during the day

File: code/client/src/components/NFTGrid.jsx (hunk 2 above)
Comment (haolinj) [reply code/client/src/components/NFTGrid.jsx]: Removed.

File: code/client/src/components/NFTGrid.jsx (hunk 3 above)
Comment (haolinj) [reply code/client/src/components/NFTGrid.jsx]: Added the wrapper style back.

**Comment by polymorpher** [comment]
looks good

**Pull request merged**